src/GitHub.Exports/Helpers/SimpleRepositoryModelExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using GitHub.Models;
+using System;
+using System.Linq;
+using System.IO;
+
+namespace GitHub.Extensions
+{
+    using VisualStudio;
+
+    public static class SimpleRepositoryModelExtensions
+    {
+        public static bool HasCommits(this ISimpleRepositoryModel repository)
+        {
+            var repo = Services.GetRepoFromPath(repository.LocalPath);
+            return repo?.Commits.Count() > 0;
+        }
+        public static bool HasContent(this ISimpleRepositoryModel repository)
+        {
+            var dir = new DirectoryInfo(repository.LocalPath);
+            return dir.EnumerateFileSystemInfos("*", SearchOption.TopDirectoryOnly)
+                      .Any(x => ((x.Attributes.HasFlag(FileAttributes.Directory) || x.Attributes.HasFlag(FileAttributes.Normal)) &&
+                                !x.Name.StartsWith(".", StringComparison.Ordinal) && !x.Name.StartsWith("readme", StringComparison.OrdinalIgnoreCase)));
```

Comment:
What if there's a file named `readme-but-not-a-readme.md`? Certainly an edge case and maybe we don't care. But I think a safer check is `!x.Name.StartsWith("readme.") && !x.Name.Equals("readme", StringComparison.OrdinalIgnoreCase)`

Also, what's the purpose of this method? Seems like if it has a `readme`, then it _does_ have content. Maybe clarify in the method name and/or comment why we ignore those files when determining if it has content.